userguide/content/en/docs/Adding content/navigation.md

```diff
@@ -94,6 +94,16 @@ You can find a complete list of icons to use in the [FontAwesome documentation](
 
 Out of the box, if you want to use icons, you should define icons for all items on the same menu level in order to ensure an appropriate look. If the icons are used in a different way, individual CSS adjustments are likely necessary.
 
+### Add manual links to section menu
+
+By default the section menu is entirely generated from your section's pages. If you want to add a manual link to this menu, such as a link to an external site or a page in a different section of your site, you can do this by creating a placeholder page file in the doc hierarchy with the appropriate weight and some special parameters in its metadata (frontmatter). If a page has defined the parameter `manualLink` in its metadata, the generated link in the section menu for this page will be replaced by the value of `manualLink`. You can optionally also set a link title and target with the parameters `manualLinkTitle` and `manualLinkTarget` - for example if you want an external link to open in a new tab you can set the link target to `_blank`. 
+
```

Comment:
More out of interest, is there a reason why the user would set the link title, I've only ever seen that used for a tooltip thing on a link?  I presume the actual link text in the menu is the page title.

(also should we mention explicitly in the docs that the `noopener` is added automatically for new tabs so you don't run the risk of weird tab hijacks?)

Reply:
The assignment of meaningful title attributes for links is a (quiet old) technique for improving accessibility in the WEB. Nowadays it would be even better to use the various aria-* elements correctly. But I think it makes sense to offer a way to set at least the title attribute.

If desired, I can also leave it out or only set the title by default if the title and linkTitle differ in the front matter.
What do you think?

Yes, a info about the automatically set `noopener` would be fine.

Reply:
No, I think giving the option to the user is good. I'll update the bit about `noopener`.

Reply:
To clarify: Currently the link title is always set, either with the manualLinkTitle or the title from the frontmatter. It might make more sense to only set the title if it was set manually or at least in the front matter the title deviates from the linkTitle (which is the visible text in the menu).

Reply:
Since my last commit (https://github.com/google/docsy/pull/511/commits/7273745ae1b1aebe151190d4763fe6899f9f7a69) the title attribute of a link in the section menu and section index will only get set, if manualLinkTitle is set or the shown link text differs
from the title of a page. Which (I think / hope) should be the current recommendation.

In the commit (https://github.com/google/docsy/pull/511/commits/efb434a5ab3ab79819b4cf74e2b808b49ea0971e) I've added a note, that also in the section index the links will be overwritten by manualLink.